packages/appium/test/parser-specs.js

```diff
@@ -60,14 +60,14 @@ describe('Server Parser', function () {
     (() => {p.parse_args(['-dc', 'does/not/exist.json']);}).should.throw();
   });
   it('should parse --allow-insecure correctly', function () {
-    p.parse_args([]).allowInsecure.should.eql([]);
+    p.parse_args([]).should.have.property('allowInsecure', undefined);
```

Comment:
does the parser in base driver code still work as expected if undefined is passed instead of an empty array?

Reply:
how would you pass in `undefined`?  you mean just calling `parse_args()`?